src/css/ProjectsPage.css

```diff
@@ -0,0 +1,89 @@
+.projects-page {
+    padding: 2rem;
+  }
+  
+  .projects-grid {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+    gap: 2rem;
+    margin-bottom: 4rem;
+  }
+  
+  .project-card {
+    background: #fff;
+    border-radius: 15px;
+    box-shadow: 0 3px 10px rgba(0,0,0,0.1);
+    padding: 1rem;
+    text-align: center;
+    transition: transform 0.2s;
+  }
+  
+  .project-card:hover {
+    transform: translateY(-5px);
+  }
+  
+  .project-card-image img {
+    max-height: 150px;
+    object-fit: contain;
+    margin-bottom: 1rem;
+  }
+  
+  .project-card h3 {
+    font-size: 1.2rem;
+    margin: 0.5rem 0;
+  }
+  
+  .project-card p {
```

Comment:
Centering the paragraph text feels awkward. That is especially for the cards that have longer paragraphs. I tried `justify` but it was too spastic and awkward. `text-align: left` sits comfortably. I think this will also set us up to position a project icon to the left of each project name. I am also imaging a short list of custom badges at the bottom right.